### DISCUSSION TITLE
word2016版本下自动图文集里面的内容会消失

### DISCUSSION THREAD

**Discussion by Huoyucat** [sikouhjw/gdutthesis-Word#5] [Q&A]
一开始使用是没什么问题的，但是过了一段时间再去看图文集里面的内容就会消失

**Comment by sikouhjw** [comment]
不清楚，我没遇到这个问题，自动图文集的内容是保存在模板里的，不应该出现这个问题。

你试试一开始的时候输出自动图文集的内容，然后自己手动再保存到自动图文集一次？

**Comment by sikouhjw** [comment]
![图片](https://user-images.githubusercontent.com/49770925/157597894-eabf76d6-19b4-4262-ab59-104b169143aa.png)

自动图文集保存位置是 `dotx` 文件，平时新建的 Word 文件是从 `Normal.dotm` 创建出来的，如果能保存在 `Normal.dotm` 并且新建 `docx` 后正常使用，那本模板也不应该出问题，可能是你把 `dotx` 文件删除了？
